wsimod/nodes/nodes.py

```diff
@@ -1358,6 +1412,22 @@ def __init__(self, decays={}, parent=None, number_of_timesteps=1, **kwargs):
 
         self.end_timestep = self._end_timestep
 
+    def apply_overrides(self, overrides: Dict[str, Any] = {}):
+        """Apply overrides to the decayqueuetank.
+
+        Enables a user to override any of the following parameters:
+        number_of_timesteps, decays.
+
+        Args:
+            overrides (dict, optional): Dictionary of overrides. Defaults to {}.
+        """
+        self.number_of_timesteps = overrides.pop(
+            "number_of_timesteps", self.number_of_timesteps
+        )
+        self.internal_arc.number_of_timesteps = self.number_of_timesteps
+        self.internal_arc.decays.update(overrides.pop("decays", {}))
```

Comment:
For consistency with using the pre-specified number of timesteps?

```suggestion
        self.internal_arc.decays.update(overrides.pop("decays", self.internal_arc.decays))
```